### ISSUE TITLE
Exchange: Cannot send mail error Test Authentication needed

### ISSUE THREAD

**Issue by lumix1991** [Airmail/Airmail_Public#661]
Since a few days i am using Airmail 2 again, i added 3 Exchange mailboxes.

During my job today there were no problems at all, but now i have the following error on all mailboxes when i am sending mail;
https://www.dropbox.com/s/xgjmjn390hpcpkt/Screenshot%202015-01-12%2020.03.44.png?dl=0

When i restart my Mac (or Airmail) nothing changed, this error will remain. In outlook (which i have also installed on my Mac) everything is working as it should.

Can you fix this?

**Comment by honu-girl** [comment]
Same issue here, and have had it repeatedly, on both the Beta and public version. 

Really weird thing for me is that things will work happily, sometimes for days, and then with nothing obvious changing (e.g., I don't log out, the server remains up, Airmail isn't closed and reopened) it will stop working. May not work for a few minutes, a few hours, or a few days, and then, again without anything obvious on my part - it starts working again.

If someone could guide me on what I should track or monitor to try to capture what causes this, I'd be happy to do it. I'd love to solve this so I don't have to use multiple mail programs :(

![error](https://cloud.githubusercontent.com/assets/5091314/5759283/517a5718-9c92-11e4-943f-2e89103cad67.jpg)

**Comment by honu-girl** [comment]
Same issue here, repeatedly, on both Beta and public version, for several versions. It used to only happen when I was within the same domain (e.g., I have an exchange account for work, and I'd only have this issues AT work, but not with Airmail at home). But now I've been having the issue at work and outside of work. This makes Airmail unusable for me - any hope for a solution?

**Comment by andreasg** [comment]
I'm encountering this issue as well on my Exchange account. It seems to be more frequent on emails with attachments.

Really hoping for a fix!

**Comment by neW1-** [comment]
I am on an exchange account as well but it happens with any message regardless of weather it has an attachment or not.

**Comment by RuthieF** [comment]
Same issue here! Does not matter if attachement or not just randomly I am not able to use the exchange account at all due to this error. I will only sometimes notice when my inbox has been too quiet and sure enough I try to send a mail and get this message. It is begining to happen more and more frequently! Five times in the last three months, it's basically rendering this application unusable.

**Comment by adamdrake** [comment]
Same problem for me, and it seems I'm more or less completely unable to send messages.  The only workaround seems to be closing, reopening, then opening the message in the Drafts directory and clicking send again.

**Comment by lumix1991** [comment]
Why is there no response from the Airmail staff, this is a huge problem and Airmail is not usable for us when this problem remains.

We payed money for this super app, but with this problems it's useless.

**Comment by RuthieF** [comment]
Agreed. I liked Airmail but I had to just delete it. I can not afford to have HOURS where my email does not work. Annoyed I paid for an app with no real tech support so the product is basically useless to me. They should be paying attention to this thread as the Exchange issue is a real problem.

- Fed Up

**Comment by Gertdus** [comment]
I really find this error very frustrating. It is completely unclear for me what causes this. As others mentioned at the moment the product is pretty much useless...

**Comment by sforseil** [comment]
I sincerely regret having paid for the app, it is completely useless without the Exchange protocol fixed.

**Comment by mattsson** [comment]
Happening here regularly as well.

**Comment by laantorchaweb** [comment]
I'm having the same problem here, any news about if this is going to be fixed?

**Comment by lumix1991** [comment]
I send a e-mail to the people of Airmail and told them the existence of this Github Issue, hopefully they will check this and solve the problem!

The app is still useless for me, what a shame! It's such a nice app!

**Comment by lukerollans** [comment]
I have two Exchange accounts in Airmail (the second has only just been added). The first account has worked fine for months, the second cannot send.

Weird issue

**Comment by Bloophub** [comment]
please try the latest beta Please check the latest beta, http://airmailapp.com/beta

**Comment by dangetman** [comment]
For exchange account I'm getting the following error

"The settings are not valid: Authentication needed"

I have never used Airmail before, but I'm using the same settings for the default mail client on the mac with no issues.

Any solutions for this?

**Comment by darren-james** [comment]
Same issue for me too. Was working, as of yesterday it is no longer working. Deleted account and tried to create again results in the same failure.

**Comment by lumix1991** [comment]
They closed this issue but it isn't resolved. 

I was using Outlook in the mean time, and installed Airmail (latest version) yesterday. Same problems.

Can they please re-open this issue and solve the problems?

**Comment by dsibilly** [comment]
Just resumed using Airmail 2 with the 2.6.1 update a few days ago, and this problem has recurred. Apple Mail and Outlook work just fine, but Airmail 2.6.1 on OS X 10.11.3 suddenly stopped working yesterday.

This issue should be re-opened.

**Comment by bobbywlindsey** [comment]
Having the same problem with Exchange. Issue needs to be re-opened.

**Comment by rafo** [comment]
I have the same problem with Exchange. Tried everything - no help. It gives me the feeling that this App is not usable at a professional level...

**Comment by rafo** [comment]
Maybe there is a problem with multiple Exchange Accounts?

**Comment by lumix1991** [comment]
I send them an email to ask them to re-open this issue. Lets see....

**Comment by vloris** [comment]
I have this issue too today.
Closed airmail, (in an attempt to send it with Apple Mail), but when I reopened Airmail, it magically tried again to send the failed messages in my Drafts folder and succeeded!

**Comment by hanche** [comment]
Glad to see this issue reopened. It bites me too, at unpredictable intervals. The problem often persists for days before it disappears as mysteriously as it appeared. Currently Airmail on my work laptop is unusable due to this, so I use my private laptop for all work email. I clearly cannot recommend Airmail to my colleagues until this bug is well and truly squashed!

**Comment by manuelkuhs** [comment]
Strange, yesterday I updated beta to 352 and my (single) Exchange account still worked fine.

Suddenly today Airmail tells me my beta is out-of-date. So I download the same (352) again. Suddenly, Exchange doesn't work anymore with exact symptoms as described in this thread.

**Comment by manuelkuhs** [comment]
IT WORKS!

I removed my Exchange account. Reinstalled the app. Added the account - now it works. Beta 352.

**Comment by mezkalito** [comment]
It does still not work to me.

**Comment by mezkalito** [comment]
I installed the Beta 354, but after 6 hours the problem appears again.

**Comment by jmobiata** [comment]
Same issue. Apple Mail has no issues with same Exchange settings. Messages constantly fail on Airmail.

**Comment by Bloophub** [comment]
try to enable legacy sender

**Comment by tralafiti** [comment]
> try to enable legacy sender
@Bloophub 

Did not help. Exchange account unusable and even removing and re-adding the account does not help. Had this problem two times before and usually it vanishes after days or weeks just as magically as it appeared.

**Comment by brocke** [comment]
Same issue over here as well! Again, Apple Mail has NO issues with exchange, Airmail 3 has! Without identifiable reason Airmail suddenly stops working.

**Comment by jmobiata** [comment]
Sorry, done with Airmail now. I can't risk having failed messages languish in my drafts folder. No other mail program on Mac has this issue.

**Comment by Dutchmen157** [comment]
So happy this issue has been re-opened. Airmail has worked fine on both my old laptop, new laptop and old iMac but on my new iMac at work it i have been getting the exchange test failed etc error for months. hasn't worked once... IT tried to fix and Airmail did not get back to them with the ticket. Honestly it sucks because i was hoping airmail 3 would fix this and nope it hasn't at all. Im stuck using mac mail which is painful. How has this not been addressed after a full year of this being a known issue...PLEASE FIX THIS.

**Comment by moesphemie** [comment]
So i've been chatting with a support-dude and he told me that Airmail does not support ActiveSync… This could be a possible problem (In my case it is since our EWS is working with ActiveSync)

**Comment by bentosquares** [comment]
Because Airmail doesn't use ActiveSync, it uses EWS as @moesphemie mentioned. I don't fully understand why this doesn't seem to be an issue with some setups when outside a company network but problematic within the company; but for our setup I had to enable "Basic Authentication" for the EWS virtual directory in the Exchange Control Panel (initially only "Integrated Windows Authentication" was configured). That would be the first thing I would mention to Exchange Admins when troubleshooting the "Authentication Needed" error, after the usual DNS resolution and SSL checks.

**Comment by moesphemie** [comment]
@bentosquares i've talked to our admins but all they said is that i should switch to a windows-computer in order to get outlook work good… F*ckd me up and now i'm back to good old mail.app…

**Comment by rafo** [comment]
@moesphemie  Just the same with me...

**Comment by Dutchmen157** [comment]
it can't be a network/business issue i don't think. Example - my old work mac had airmail running fine. My new work mac can't get the exchange sync to work - 'Authentication needed' etc issue. It has to be an Airmail issue not a network one. Other people in my office on macs have airmail running fine no problem - it just seems to be a random bug that Airmail can't be bothered to try and fix...

**Comment by Bloophub** [comment]
Please check:
- Antivirus
- Firewall, little snitch or similae
- something could blocking the 443 port
- cookies  
- Bartender app or SYML (breaking sandbox)
... something else on your setup
